charts/capsule/templates/post-install-job.yaml

```diff
@@ -25,6 +25,10 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name | quote }}
         helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     spec:
+      {{- with .Values.imagePullSecrets }}
```

Comment:
Should we use the Manager pull secrets here, or maybe, would be better to provide different secrets just for this Job, such as `jobs.image.imagePullSecrets`?

Reply:
I'm doing a lot of stuff with charts at the moment - most places I see it just defined at global level for all images in the chart. Other charts like [bitnami/external-dns](https://github.com/bitnami/charts/blob/master/bitnami/external-dns/values.yaml) use a global value as well as individual values.

Either works for me to be honest - I'd be tempted to just use the main imagePullSecrets for simplicity's sake, and if a use case comes about where it's better to specify it separately for the jobs, then add it then, but if you think it's better to separate it from the get-go just let me know and I'll do that.

Reply:
No problem, it was just to start the discussion.

Let's implement this way, we have always room for improvement :)